File: packages/react-native/temp/package/ReactAndroid/src/main/java/com/facebook/react/views/textinput/ScrollWatcher.java
/*
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

package com.facebook.react.views.textinput;

public interface ScrollWatcher {
  void onScrollChanged(int horiz, int vert, int oldHoriz, int oldVert);
}

File: packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/textinput/ScrollWatcher.kt
/*
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

package com.facebook.react.views.textinput

public interface ScrollWatcher {
  public fun onScrollChanged(horiz: Int, vert: Int, oldHoriz: Int, oldVert: Int): Unit
}
